rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
+gem 'rails', path: '/rails'
+gem 'ffaker'
+gem 'sqlite3'
+gem 'stackprof'
+gem 'mysql2'
+gem 'pg'
+gem 'arel', github: 'rails/arel', branch: 'master'
```

Comment:
there are 2 things we are trying to test 

1. Is rails getting faster / slower ... every commit ... in which testing master is appropriate. 
2. Is ruby getting faster / slower ... every commit ... in which we must pin a version